### DISCUSSION TITLE
Customize Default Shortcuts

### DISCUSSION THREAD

**Discussion by omerharuncetin** [Milkdown/milkdown#916] [Q&A]
Hello,

Currently, I am using version `6.3.0` for `milkdown`. 

I want to change the behavior of `Enter` and `Shift-Enter` or maybe the other default keys in the future.

Do you know how I can do that?

**Comment by sarveshpop** [comment]
@omerharuncetin I'm looking to do the same. Did you figure out a way to do this?
